<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#centralwidget{
background-color:#FFF;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="search_city">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>90</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Sarch</string>
    </property>
   </widget>
   <widget class="QWidget" name="weatherWidget" native="true">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>210</y>
      <width>300</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#weatherWidget{

background-color: rgb(12, 97, 255);
border-radius:20px;
}</string>
    </property>
    <widget class="QLabel" name="viewIcon">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel#viewIcon{

border-radius:20px;

}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>weatherIcons2/04n.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="temperature">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>80</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>24 Â°C</string>
     </property>
    </widget>
    <widget class="QLabel" name="weather">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>70</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>weather</string>
     </property>
    </widget>
    <widget class="QLabel" name="city_country">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>110</y>
       <width>161</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>City, Country</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
